### PR TITLE
show list of requests on home screen

### PR DESCRIPTION
**Pull request by bluquar**
https://user-images.githubusercontent.com/4309735/143659103-bb3b420b-3993-47e8-b2c7-cab6b874cc01.mov

### CODE REVIEW

File: screens/HomeScreen.tsx [hunk 1]
```diff
@@ -1,27 +1,102 @@
...
+          <Paragraph>For: User-7</Paragraph>
+          <Paragraph>Created by: User-13</Paragraph>
+          <Paragraph>Where: Zip code {zipCode}</Paragraph>
+          <Paragraph>
+            Created <TimeAgo when={new Date(createdAt)} />
+          </Paragraph>
+          <Paragraph>Working on this: No one yet</Paragraph>
+        </Card.Content>
+        <Card.Actions>
+          <AidButton
```
Comment (lowellbander) [inline screens/HomeScreen.tsx:77]: Why this component Aid prefixed but not the time component?

**Review by lowellbander — APPROVED** [review]
FUCK YES #itshappeningronpaul

File: screens/HomeScreen.tsx [hunk 2]
```diff
@@ -1,27 +1,102 @@
...
+      id
+      whatIsNeeded
+      completed
+      zipCode
+      createdAt
+    }
+  }
+`;
+
+type ListItemData = {
```
[the remark this answers is not in the document]
Comment (bluquar) [reply screens/HomeScreen.tsx:24]: Not sure! Tracking in #39

File: screens/HomeScreen.tsx [hunk 3]
```diff
@@ -1,27 +1,102 @@
...
+
+  if (loading) {
+    return <ActivityIndicator />;
+  }
+
   return (
     <View style={styles.container}>
-      <Text>Home</Text>
+      <FlatList
+        data={(data as QueryResponse).allAidRequests}
```
[the remark this answers is not in the document]
Comment (bluquar) [reply screens/HomeScreen.tsx:48]: It's TypeScript :)

File: screens/HomeScreen.tsx (hunk 1 above)
Comment (bluquar) [reply screens/HomeScreen.tsx:77]: No reason!

**Pull request merged**